charts/vector/templates/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{- include "vector.labels" . | nindent 4 }}
 spec:
-  {{- if not .Values.autoscaling.enabled }}
+  {{- if and (not .Values.autoscaling.enabled) (not .Values.keda.enabled) }}
```

Comment:
I think this should be an `or`? If either of them is enabled we don't want to include the replicas key.

Reply:
As far as i understand `or` won't work here. I found that under the hood helm [uses go template lib](https://pkg.go.dev/text/template#hdr-Functions) , as stated [here](https://github.com/helm/helm/issues/6250#issuecomment-523134350). So `and` should be read as `if x then y else x.` . Here is small example how it works - https://go.dev/play/p/kDiNqowjId4 . I've also checked it with `helm install --dry-run` and verified that there is no replicas in either way.